cmd/aws/main.go

```diff
@@ -54,13 +54,14 @@ var (
 	origin                     = flag.String("origin", "", "Origin of the log, for checkpoints and the monitoring prefix.")
 	bucket                     = flag.String("bucket", "", "Name of the bucket to store the log in.")
 	dbName                     = flag.String("db_name", "", "AuroraDB name")
+	antispamDBName             = flag.String("antispam_db_name", "", "AuroraDB antispam name")
 	dbHost                     = flag.String("db_host", "", "AuroraDB host")
 	dbPort                     = flag.Int("db_port", 3306, "AuroraDB port")
 	dbUser                     = flag.String("db_user", "", "AuroraDB user")
 	dbPassword                 = flag.String("db_password", "", "AuroraDB password")
 	dbMaxConns                 = flag.Int("db_max_conns", 0, "Maximum connections to the database, defaults to 0, i.e unlimited")
 	dbMaxIdle                  = flag.Int("db_max_idle_conns", 2, "Maximum idle database connections in the connection pool, defaults to 2")
-	dedupPath                  = flag.String("dedup_path", "", "Path to the deduplication database.")
+	inMemoryAntispamCacheSize  = flag.Uint("inmemory_antispam_cache_size", 2<<10, "Maximum number of entries to keep in the in-memory antispam cache.")
```

Comment:
'2048' might well be enough, but I'd be tempted default to a larger size which can be tuned down later/via flag if need be.
I've been using 256k as a place-holder in the conformance bins (32+8 bytes of data ==> 10MB of raw cached data plus whatever overhead Go's `map` incurs)